clusterloader2/pkg/config/cluster.go

```diff
@@ -16,15 +16,18 @@ limitations under the License.
 
 package config
 
+import (
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
 // ClusterLoaderConfig represents all flags used by CLusterLoader
 type ClusterLoaderConfig struct {
-	ClusterConfig            ClusterConfig `json: clusterConfig`
-	ReportDir                string        `json: reportDir`
-	EnablePrometheusServer   bool          `json: enablePrometheusServer`
-	EnableExecService        bool          `json: enableExecService`
-	TearDownPrometheusServer bool          `json: tearDownPrometheusServer`
-	TestConfigPath           string        `json: testConfigPath`
-	TestOverridesPath        []string      `json: testOverrides`
+	ClusterConfig            ClusterConfig    `json: clusterConfig`
+	ReportDir                string           `json: reportDir`
+	EnablePrometheusServer   bool             `json: enablePrometheusServer`
+	EnableExecService        bool             `json: enableExecService`
+	TearDownPrometheusServer bool             `json: tearDownPrometheusServer`
+	TestScenario             api.TestScenario `json: testScenario`
```

Comment:
Can we remove TestOverridesPath now?

Reply:
Yes, it's being removed (or rather moved to test scenario structure) by this PR.